client/verta/verta/data_types/_discrete_histogram.py

```diff
@@ -105,6 +105,9 @@ def diff(self, other):
         )
 
     def normalized_data(self, keys=None):
-        total = sum(self._data)
         keys = keys if keys else self._buckets
+        total = sum(self._data)
```

Comment:
Actually, @daniel-verta Should this be `sum(data)`, using the `data = [self._data_dict[k] ...]` from the line below?

Reply:
I think you're right that it should actually be `total = sum(data)` after the definition of `data = ...`